compiler/rustc_builtin_macros/src/errors.rs

```diff
@@ -845,6 +845,17 @@ pub(crate) struct AsmOptAlreadyprovided {
     pub(crate) full_span: Span,
 }
 
+#[derive(Diagnostic)]
+#[diag(builtin_macros_global_asm_unsupported_option)]
+pub(crate) struct GlobalAsmUnsupportedOption {
+    #[primary_span]
+    #[label]
+    pub(crate) span: Span,
+    pub(crate) symbol: Symbol,
+    #[suggestion(code = "", applicability = "machine-applicable", style = "tool-only")]
```

Comment:
I think anything machine applicable needs a `//@ run-rustfix` test to make sure the output is correct